openapi/index.yaml

```diff
@@ -34,6 +34,13 @@ info:
     sent to the fake Fiscal Code will be notified to the email address used
     during the registration process on the developer portal.
 
+    # Rate limits
+
+    All API requests are subject to rate limits.
+    Rate limits apply on a subscription key basis.
+    The current rate limit is of `150 requests` every `5 seconds`.
+    If limits are exceeded the API server returns a 429 (Too Many Requests) response.
```

Comment:
do we return some headers to the caller ? (ie x-retry-after)

Reply:
Not for now, but we can add both retry-after and remaining-calls count as headers.

Reply:
ok (so we will add those headers here when ready 👍 )

Reply:
do we return the headers now? :)

Reply:
ping :)